### PR TITLE
Blacklists guardians from using fugu glands

### LINKED ISSUE #7287: Holoparas can use Fugu Glands
## Round ID:
None

## Testmerges:

None
## Reproduction:

I think being able to use fugu glands on holoparas is unintended, and should be fixed

Reproducting is as follows:
1. Get a fugu gland
2. Get a holopara
3. Feed the fugu gland to the holopara

### PR DESCRIPTION
**Pull request by dwasint**
## About The Pull Request
closes https://github.com/BeeStation/BeeStation-Hornet/issues/7287

## Why It's Good For The Game
Fugu Glands double a targets damage, multiply its hp by 1.5, lets it smash rwalls. Its not balanced around guardians and should probably be removed

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
tweak: holo parasites can't use fugu glands anymore to double their damage
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm [hunk 1]
```diff
@@ -123,7 +123,7 @@
 
 /obj/item/fugu_gland/afterattack(atom/target, mob/user, proximity_flag)
 	. = ..()
-	if(proximity_flag && isanimal(target))
+	if(proximity_flag && isanimal(target) && !isguardian(target))
```
Comment (Sarchutar) [inline code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm]: Why not just use the `banned_mobs` list fugu glands have instead of doing this?

Comment (dwasint) [reply code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm]: fair assesment i thought guardians were a branched type of things like they use to be with dexterous

Comment (dwasint) [reply code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm]: fixed in latest commit

**Review by PowerfulBacon — APPROVED** [review]
Wait so this var was just unused before? thats weird

**Pull request merged**